.github/workflows/cd-sql-engine-tests.yaml

```diff
@@ -91,6 +91,25 @@ jobs:
           additional-pytest-options: ${{ env.ADDITIONAL_PYTEST_OPTIONS }}
           make-target: "test-databricks"
 
+   trino-tests:
+    environment: DW_INTEGRATION_TESTS
+    name: Trino Tests
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'Run Tests With Other SQL Engines' }}
+    runs-on: ubuntu-latest
```

Comment:
I'm pretty sure we'll need to add a service container here:

https://docs.github.com/en/actions/using-containerized-services/about-service-containers

Reply:
Just to be clear, I will deal with this since you don't have access to our CI runners or permission to kick off jobs.